pkg/injector/envoy_config.go

```diff
@@ -198,3 +208,21 @@ func getXdsCluster(config envoyBootstrapConfigMeta) map[string]interface{} {
 		},
 	}
 }
+
+func (wh *mutatingWebhook) createOrUpdateWasmConfigMap(namespace string) (*corev1.ConfigMap, error) {
```

Comment:
Document this at https://github.com/openservicemesh/osm/blob/main/docs/content/docs/uninstallation_guide.md#remaining-after-osm-uninstallation?